File: INICIANTE/1066/main.rb
# frozen_string_literal:true

def formatted_print(evens, positives, negatives)
  puts "#{evens} valor(es) par(es)"
  puts "#{5 - evens} valor(es) impar(es)"
  puts "#{positives} valor(es) positivo(s)"
  puts "#{negatives} valor(es) negativo(s)"
end

evens = 0
positives = 0
negatives = 0
5.times do
  number = gets.to_i
  evens += 1 if number.even?
  positives += 1 if number.positive?
  negatives += 1 if number.negative?
end
formatted_print(evens, positives, negatives)

File: INICIANTE/1066/main.dart
import 'dart:io';

void formattedPrint(int evens, int positives, int negatives) {
  print('$evens valor(es) par(es)');
  print('${5 - evens} valor(es) impar(es)');
  print('$positives valor(es) positivo(s)');
  print('$negatives valor(es) negativo(s)');
}

void main() {
  int evens = 0;
  int positives = 0;
  int negatives = 0;
  for (int times = 0; times < 5; times++) {
    int number = int.parse(stdin.readLineSync()!);
    if (number % 2 == 0) evens++;
    if (number > 0) positives++;
    if (number < 0) negatives++;
  }
  formattedPrint(evens, positives, negatives);
}
